.golangci.yml

```diff
@@ -108,8 +108,11 @@ issues:
     - path: internal/martian
       linters:
         - goheader
-        - errcheck
```

Comment:
This does not belong in that commit.

Reply:
Some rebasing must have gone wrong. Good catch